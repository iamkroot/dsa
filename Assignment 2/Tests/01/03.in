3
0 10 4 
1 0 0 
0 4 0 
