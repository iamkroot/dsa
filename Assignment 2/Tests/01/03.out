Cycles with minimum weight 9 are:
0->2->1->0
